src/main/java/io/github/thebusybiscuit/slimefun4/api/events/ClimbingPickLaunchEvent.java

```diff
@@ -0,0 +1,82 @@
+package io.github.thebusybiscuit.slimefun4.api.events;
+
+import org.apache.commons.lang.Validate;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.bukkit.event.player.PlayerEvent;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.util.Vector;
+
+import io.github.thebusybiscuit.slimefun4.implementation.items.tools.ClimbingPick;
+
+/**
+ * An {@link Event} that is called whenever a {@link Player} has
+ * used a {@link ClimbingPick} on a climbable surface.
+ *
+ * @author Linox
+ *
+ * @see ClimbingPick
+ *
+ */
+public class ClimbingPickLaunchEvent extends PlayerEvent implements Cancellable {
+
+    private static final HandlerList handlers = new HandlerList();
+
+    private Vector velocity;
+    private final ClimbingPick pick;
+    private boolean cancelled;
+
+    public ClimbingPickLaunchEvent(Player player, Vector velocity, ClimbingPick pick) {
```

Comment:
The ItemStack and Block of this Event may be good to have too.

Reply:
ItemStack is ClimbingPick anyway?

Reply:
No, ClimbingPick is a SlimefunItem, not the actual ItemStack that was used.